apps/ejabberd/test/ejabberd_sm_SUITE.erl

```diff
@@ -203,8 +205,31 @@ kv_can_be_updated_for_session(C) ->
     when_session_info_stored(U, S, R, {key2, override}),
 
     [#session{sid = Sid, info = [{key1, val1}, {key2, override}]}]
-     = ?B(C):get_sessions(U,S).
-
+     = ?B(C):get_sessions(U, S).
+
+cannot_reproduce_race_condition_in_store_info(C) ->
```

Comment:
I finally decided not to move this single test case to a separate group. It's just addition to `mnesia` group.